taichi/ir/expr.cpp

```diff
@@ -179,7 +179,9 @@ Expr load(const Expr &ptr) {
 Expr ptr_if_global(const Expr &var) {
   if (var.is<GlobalVariableExpression>()) {
     // singleton global variable
-    TI_ASSERT(var.snode()->num_active_indices == 0);
+    TI_ASSERT_INFO(
+        var.snode()->num_active_indices == 0,
+        "Please always use 'x[None]' (instead of simply 'x') to access any 0-D tensor."
```

Comment:
```suggestion
        "Please always use 'x[None]' (instead of simply 'x') to access any 0-D tensor.");
```

Reply:
Sorry for merging without proofread! Did a OFT fix in #1356.